changelog.d/8887.bugfix

```diff
@@ -0,0 +1 @@
+Add X-Robots-Tag header to stop web crawlers from indexing media.
```

Comment:
I think this is more a change (`.misc`) rather than fixing a bug.

Reply:
This is probably fairly user visible so shouldn't be `misc` IMO.

Reply:
Hm, good point.

Reply:
Could also be a feature